tuna/rocmlir/rocmlir_worker.py

```diff
@@ -170,17 +173,21 @@ def run_cmd(self):
       special_args = "--operation conv"
     elif self.dbt.config_type == ConfigType.gemm:
       special_args = "--operation gemm"
+    elif self.dbt.config_type == ConfigType.attention:
+      special_args = "--operation attention --verify-mode none"
     else:
       raise ValueError(f"Config type {self.dbt.config_type} not yet supported.")
+    if self.dbt.session.tuning_space:
+      special_args += f" --tuning-space={self.dbt.session.tuning_space.name}"
 
     if not os.path.exists("./bin/tuningRunner.py"):
       raise FileNotFoundError("tuningRunner.py not found;"
                               "  wrong directory or missing setup")
 
     cmd = env_str + f" python3 ./bin/tuningRunner.py -q {special_args} \
                      --config='{config_string}' --mlir-build-dir `pwd` \
-                     --output={self.output_filename()} --tflops \
-                     --rocmlir_gen_flags='--device={self.gpu_id}'"
+                     --output=- --tflops \
```

Comment:
'-' represents stdout, which tuningRunner.py now supports for its output.  Safer than writing to a file because we don't need to add hoops to avoid collisions.  Redirect stderr out of the way, because otherwise it confuses things.